magefile.go

```diff
@@ -105,25 +105,24 @@ func Integration() error {
 }
 
 // Spec generates an OpenAPI spec yaml based on code annotations.
-func Spec(fmt bool) error {
+func Spec() error {
 	swagCommand := "swag"
 	if err := installIfNotPresent(swagCommand, "github.com/swaggo/swag/v2/cmd/swag@v2.0.0-rc3"); err != nil {
 		logrus.WithError(err).Error("failed to install swag")
 		return err
 	}
 
-	if fmt {
-		if err := sh.Run(swagCommand, "fmt", "-d", "pkg/server/router"); err != nil {
-			logrus.WithError(err).Error("failed to format swagger docs")
-			return err
-		}
+	if err := sh.Run(swagCommand, "fmt", "-d", "pkg/server/router"); err != nil {
+		logrus.WithError(err).Error("failed to format swagger docs")
+		return err
 	}
+
 	// One of the dependencies we have (antlr4) does not play nicely with swaggo, but we need to enable dependencies
 	// because many of our external API objects have ssi-sdk objects. We can work around this by setting depth. You can
 	// see a discussion of this topic in https://github.com/swaggo/swag/issues/948.
 	// We also set parseGoList because it's the only way parseDepth works until the following is fixed:
 	// https://github.com/swaggo/swag/issues/1269
-	return sh.Run(swagCommand, "init", "-g", "cmd/ssiservice/main.go", "--pd", "-o", "doc", "-ot", "go,yaml", "--parseInternal")
+	return sh.Run(swagCommand, "init", "-g", "cmd/ssiservice/main.go", "--overridesFile", "doc/overrides.swaggo", "--pd", "--parseInternal", "-o", "doc", "-ot", "go,yaml")
```

Comment:
If the comment above doesn't apply anymore can you update please?

Reply:
done

Reply:
done